Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/layout/AutofitGridLayoutManager.kt

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.layout
+
+import android.content.Context
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.recyclerview.widget.RecyclerView.Recycler
+import org.dolphinemu.dolphinemu.R
+
+/*
+*  Cut down version of the solution provided here
+*  https://stackoverflow.com/questions/26666143/recyclerview-gridlayoutmanager-how-to-auto-detect-span-count
+*/
```

Comment:
This should either use `//` comments as per Dolphin policy, or should be a javadoc comment.